dbt_subprojects/dex/models/trades/ronin/platforms/tamadotmeme_v1_ronin_base_trades.sql

```diff
@@ -0,0 +1,150 @@
+{{ config(
+    schema = 'tamadotmeme_v1_ronin',
+    alias = 'base_trades',
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['tx_hash', 'evt_index']
+) }}
+
+-- Retrieve hourly Ronin prices for the WRON token.
+WITH ronin_price AS (
+  SELECT
+    hour,
+    median_price AS ron_price
+  FROM {{ source('dex', 'prices') }}
+  WHERE contract_address = 0xe514d9deb7966c8be0ca922de8a064264ea6bcd4
+),
```

Comment:
This may not be the correct approach to implement a `base_trades` spell,
please read the document first:
https://github.com/duneanalytics/spellbook/blob/main/dbt_subprojects/dex/models/trades/readme.md